pkg/trace/api/api.go

```diff
@@ -329,19 +308,27 @@ func (r *HTTPReceiver) tagStats(req *http.Request) *info.TagStats {
 	})
 }
 
-func (r *HTTPReceiver) decodeTraces(v Version, req *http.Request) (pb.Traces, error) {
-	if v == v01 {
+func (*HTTPReceiver) decodeTraces(v Version, req *http.Request) (pb.Traces, error) {
```

Comment:
nit: Does `decadeTraces` still need to be a method on `HTTPReceiver`? It looks like it doesn't operate on it at all.